### ISSUE TITLE
Bug: Cathy crashes after pressing continue when the level is finished

### ISSUE THREAD

**Issue by wellx3** [bunfan/beat-banger-public#70]
### Discord Tag

@omer_faruk2

### What happened?

well i 5 star the expert difficulty and after that I pressed continue and it just crashed

### Version

3.165 (Secret)

### Relevant log output

```shell
Godot Engine v4.2.dev3.official.013e8e3af - https://godotengine.org
USER ERROR: Unrecognized input string "8e4f" in mapping:
03000000790000000600000000000000,G-Shark GS-GP702,crc:8e4f,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b3,y:b0,platform:Windows,
   at: parse_mapping (core/input/input.cpp:1443)
(this is what I got after opening game again so I don't know if it has anything to do with the crash)
```

**Comment by Stefan-Mitric** [comment]
Claires Level Crashing #84

**Comment by komdog** [comment]
Dupe of fixed issue #84
